.github/workflows/deploy.yml

```diff
@@ -12,11 +12,14 @@ jobs:
       - name: Install node version
         uses: actions/setup-node@v1
         with:
-          node-version: 12.x
-      - name: Build
+          node-version: 14.x
```

Comment:
It looks good, just a small mention. Let's use node version `14` for all workflows just in case there are any inconsistencies between 14 and 14.x .